mlflow_extensions/serving/engines/vllm_engine.py

```diff
@@ -148,6 +149,11 @@ def engine_pip_reqs(
         return default_installs
 
     def _setup_snapshot(self, local_dir: str = "/root/models"):
+        if local_hf_model_path := get_local_snapshot(
```

Comment:
hmm we may want to skip this entire logic if self.cache dir is empty right? and can you also add similar logic to sglang_engine.py for tokenizer and model download.

Reply:
right here would be the code that needs to be written:
https://github.com/stikkireddy/mlflow-extensions/blob/cf8c7f98bce0582758305e31928de0ca4b8849af/mlflow_extensions/serving/engines/sglang_engine.py#L141-L153